db-migrations/src/main/resources/db/migration/V61__deltaker-delt-med-arrangor.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE deltaker_status
```

Comment:
Syns ikke det er så logisk at `er delt med arrangør`skal ligge på status. Selv om verdien vil kunne endre seg basert på statusen så er det en deltaker som er delt eller ikke, ikke selve statusen

Reply:
Kom til å tenke på noe delvis relatert også. Er det forventet at vi holder styr på historikken for om en deltaker har vært delt eller ikke